lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -373,6 +373,98 @@
         default: no
         version_added: "2.8"
 
+    entrust_cert_type:
+        description:
+            - The type of certificate product to request.
+            - This is only used by the C(entrust) provider.
+        type: str
+        default: STANDARD_SSL
+        choices: [ 'STANDARD_SSL', 'ADVANTAGE_SSL', 'UC_SSL', 'EV_SSL', 'WILDCARD_SSL', 'PRIVATE_SSL', 'PD_SSL', 'CODE_SIGNING', 'EV_CODE_SIGNING',
+        'CDS_INDIVIDUAL', 'CDS_GROUP', 'CDS_ENT_LITE', 'CDS_ENT_PRO', 'SMIME_ENT' ]
+        version_added: "2.9"
+
+    entrust_requester_email:
+        description:
+            - The email of the requester of the certificate (for tracking purposes).
+            - This is only used by the C(entrust) provider.
+            - This is required if the provider is C(entrust)
```

Comment:
```suggestion
            - This is required if the provider is C(entrust).
```
(Also applies to other occurrences.)